include/envoy/network/listener.h

```diff
@@ -43,6 +43,14 @@ class ListenerConfig {
    */
   virtual bool bindToPort() PURE;
 
+  /**
+   * @return bool if a connection should be handed off to another Listener after the original
+   *         destination address has been restored. 'true' when 'use_original_dst' flag in listener
+   *         configuration is set, false otherwise. Note that this flag will be removed from the v2
```

Comment:
can you explicitly use the word "deprecated" in here somewhere so that when we search for deprecated things this comes up?

Reply:
Done.